tests/test-schema.js

```diff
@@ -0,0 +1,142 @@
+var assert = require('chai').assert;
```

Comment:
As you're only using `assert.fail` is there a reason for using Chai over Node's own `assert` module? It has the same [`assert.fail`](https://nodejs.org/api/assert.html#assert_assert_fail_actual_expected_message_operator) method.